sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -743,11 +743,9 @@ def get_blob_client(
         :param container:
             The container that the blob is in. This can either be the name of the container,
             or an instance of ContainerProperties.
+        :paramtype container: Union[ContainerProperties, str]
```

Comment:
hmm I'm curious why we're choosing to keep the pattern for `ContainerProperties`?
While it's not currently broken that I'm aware of (no versioning here, right?) - it starts to look a bit inconsistent... and also could potentially give us future headaches if such a feature were ever introduced.

Reply:
This is a good point. There are many other examples of things like this: `ContianerProperties`, `FileProperties` (datalake and share), and likely others. We should consider those as a follow-up to this, but I think `BlobProperties` is the important one to do now since it's in a more broken state.